sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_lease_client.hpp

```diff
@@ -118,11 +124,12 @@ namespace Azure { namespace Storage { namespace Blobs {
      */
     Azure::Response<Models::BreakLeaseResult> Break(
         const BreakLeaseOptions& options = BreakLeaseOptions(),
-        const Azure::Core::Context& context = Azure::Core::Context()) const;
+        const Azure::Core::Context& context = Azure::Core::Context());
 
   private:
     Azure::Nullable<BlobClient> m_blobClient;
     Azure::Nullable<BlobContainerClient> m_blobContainerClient;
+    std::mutex m_mutex;
```

Comment:
Can the blobLeaseClient be copied/moved ?  what would it happen to the mutex in those cases?

Reply:
No, I mentioned in the email. Lease clients won't be copiable or movable

Reply:
This is the thing that concerns me the most. I guess it's OK if a pointer/reference to it can be passed to other places.

Reply:
> This is the thing that concerns me the most. I guess it's OK if a pointer/reference to it can be passed to other places.

That's correct. If customer wants to share LeaseClient, they should be using a pointer of LeaseClient.